docs/uma-tokenholders/gitHub-walkthrough.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Alternative UMIP Github Walkthrough
```

Comment:
Generally I think this page is useful, but not something we should have on the docs site. I think a better way to do this would be to just have a link on the docs site within the `UMIP Process` section, that links to a read-only public notion page or google doc.

Also a lot of this language looks like it was copied from an informal post that I made on discord. I think this should be cleaned up and formalized and only language directly relevant should be included.

Reply:
I'll delete this from this PR and look into adding to Notion / Google doc later.

Reply:
Awesome, sounds good!